### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Comment by JordanMartinez** [comment]
CI fails with this error:
```
> eslint src && spago build '--censor-lib --strict'


/home/runner/work/purescript-web-file/purescript-web-file/src/Web/File/FileReader.js
Error:   1:11  error  'FileReader' is already defined as a built-in global variable  no-redeclare

✖ 1 problem (1 error, 0 warnings)
```

**Review by kl0tl — DISMISSED** [review]
This happens because of the `/* global FileReader */` comment in src/Web/File/FileReader.js, we can remove it.

File: spago.dhall [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
```
Comment (kl0tl) [inline spago.dhall:4]: Shall we remove this comment?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: push
+on: [push, pull_request]
```
Comment (thomashoneyman) [inline .github/workflows/ci.yml:3]:
```suggestion
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

**Review by thomashoneyman — CHANGES_REQUESTED** [review]

**Comment by thomashoneyman** [comment]
We also need to update the Bower file to point to master.

**Pull request closed without merging**